tests/checkers/unittest_utils.py

```diff
@@ -400,9 +400,19 @@ def test_if_sys_guard() -> None:
 
     if sys.some_other_function > (3, 8):  #@
         pass
+
+    import six
```

Comment:
Could you also add a test like: `import six as five; if five.PY2`? This test would not pass with the current code but should pass when you're using `qname()`.